s29744-JakubMarkowski/zadania 1-14 30.09.txt

```diff
@@ -0,0 +1,86 @@
+
+Zadania wykonuje za pomocą VirtualBox.
+
+1.Uruchom terminal.
+
+
+1.prawy przycisk myszy, open in terminal.
+2.menu, terminal.
+3.sprawdziłem w internecie, jest możliwość ctrl + alt + T
+
+
+2.Wyświetl zawartość aktualnego katalogu.
+
+ls
+
+Wyświetla zawartość katalogu
+
+3.Sprawdź gdzie aktualnie się znajdujesz.
+
+pwd
+
+Wyświetla aktualną ścieżke
+
+4.Przejdź do katalogu Pulpit za pomocą ścieżki względnej
+
+cd Desktop/
+
+5.Sprawdź gdzie jesteś.
+
+pwd
+
+6.Wyświetl zawartość bieżącego katalogu.
+
+ls 
+
+7.Wyświetl zawartość swojego katalogu domowego bez przechodzenia do niego (ls z odpowiednim parametrem).
+
+ls ~
+
+przechodze do katalogu domowego
+
+8.Przejdź do katalogu Obrazy w twoim katalogu domowym za pomocą ścieżki względnej.
+
+cd ~/Music/
```

Comment:
To nie jest ścieżka względna gdyż `~` jest przez basha rozwijana na `$HOME` a ta zmienna posiada wartość ścieżki bezwględnej. Po konkatenacji z `/Music/` nadal daje ścieżkę bezwzględną.